docs/changelog.md

```diff
@@ -0,0 +1,676 @@
+# TBD
```

Comment:
Can you merge in `master` to your branch? I think this should be deleted now on `master` (since there's not really a purpose to it, since nobody's consumign the changelog since there's no artifact produced)